io/CMakeLists.txt

```diff
@@ -3,5 +3,8 @@ cxx_link(io base)
 
 add_library(file ALIAS io)
 
-cxx_test(io_test io LABELS CI)
-cxx_test(file_test io LABELS CI)
\ No newline at end of file
+if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
I did not mean to disable the whole test suite. I think it's too much.
I meant to skip specific tests inside, for example, like this:
https://github.com/dragonflydb/dragonfly/blob/62a9313a382fa737bc891d7a874fc37122b37a57/src/server/multi_test.cc#L310

same with fiber_socket_test and accept_server_test . In fact they should work.  What exactly does not work there?

Reply:
agreed. At the end, just three particular tests need to be skipped. We understood the ProcReader test, the other two tests still need investigation.